smartcontractkit/external-adapters-js
# Feature/37411 new ea metrics

## Closes #37411

## Description

......

## Changes

- Added prom-client reporting of the following metrics to core/bootstrap:
   - data_provider_request_attempts
   - data_provider_request_duration_seconds
- Updated Requester module to report the above metrics.
- Added error type label to ws_subscription_errors
- Updated redux WS functionality to report error type using the new label



## Steps to Test

1. `yarn test core/bootstrap/test/unit/metrics`
2. Run an EA locally with metrics enabled and check for new metrics & labels in the response from `/metrics` endpoint

## Quality Assurance
